pkg/products/grafana/reconciler.go

```diff
@@ -167,6 +176,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, err
 	}
 
+	if err := r.reconcileConsoleLink(ctx, client, installation); err != nil {
```

Comment:
I see we check inside the function if install type is not managed-api and call a delete rather than create it. I can't think of a case where it would be present for an rhmi installation. There might be one I'm not thinking of.

Reply:
removed delete